src/Mariuzzo/LaravelJsLocalization/LaravelJsLocalizationServiceProvider.php

```diff
@@ -64,7 +64,7 @@ public function boot()
     public function register()
     {
         // Bind the Laravel JS Localization command into the app IOC.
-        $this->app['localization.js'] = $this->app->share(function ($app) {
+        $this->app->singleton('localization.js', function ($app) {
```

Comment:
@MCMatters just verified, and I can see it there: https://laravel.com/api/5.0/Illuminate/Contracts/Container/Container.html#method_singleton